### PR TITLE
Add support for the "routing igmp-proxy" path

### PR DESCRIPTION
**Pull request by hansmi**
##### SUMMARY
Upstream documentation:
https://help.mikrotik.com/docs/display/ROS/IGMP+Proxy

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: plugins/module_utils/_api_data.py [hunk 1]
```diff
@@ -3942,6 +3942,30 @@ def join_path(path):
...
+        ),
+    ),
+    ('routing', 'igmp-proxy', 'interface'): APIData(
+        unversioned=VersionedAPIData(
+            fully_understood=True,
+            fields={
+                'alternative-subnets': KeyInfo(),
+                'comment': KeyInfo(can_disable=True, remove_value=''),
+                'disabled': KeyInfo(default=False),
+                'interface': KeyInfo(),
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:3963]: Shouldn't `interface` be the primary key here? Or can you have multiple entries per interface?

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: plugins/module_utils/_api_data.py (hunk 1 above)
Comment (hansmi) [reply plugins/module_utils/_api_data.py:3964]: Yes, you're right. Done.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@hansmi thanks!

**Pull request merged**